typescript/infra/package.json

```diff
@@ -17,7 +17,7 @@
     "@nomiclabs/hardhat-etherscan": "^3.0.3",
     "@safe-global/api-kit": "^1.1.0",
     "@safe-global/protocol-kit": "^1.0.1",
-    "@solana/web3.js": "^1.77.3",
+    "@solana/web3.js": "^1.78.0",
```

Comment:
Just updating so we can use a consistent version everywhere, which may as well be newest